### ISSUE TITLE
OBS WEBUI: User role selector on task list

### ISSUE THREAD

**Issue by Ana06** [openSUSE/open-build-service#7020]
From [openFATE](https://features.opensuse.org/323318):

> OBS Webui shall support filtering open tasks for a user by his workflow role there.
> 
> The role is defined via a group for now. Details for planing can be found in this trello card:
> 
> https://trello.com/c/H0fZCfFi/319-allow-filtering-of-requests-reviews-and-bugissues-by-user-groups
> 
> This request is just for tracking and connecting involved people.

**Comment by hennevogel** [comment]
We have /group/show for this for a while...

**Comment by coolo** [comment]
But it doesn't filter the group requests out of the others. So one group can hide all the other tasks. I have 339 tasks atm and basically no way to see what this is all about.

**Comment by hennevogel** [comment]
Yeah `User.involved_p*` still uses the users groups too. We can get rid of that of course. That would help you, not so sure about other people that are only in groups that are not overwhelming.

**Comment by coolo** [comment]
Not sure either - so I guess the feature here is to make this configurable: "support filtering"

**Comment by hennevogel** [comment]
> I guess the feature here is to make this configurable

That's cheap. Let's decide.

**Comment by coolo** [comment]
If I had to chose, I would pick github's grouping per repo. I.e. not one table to rule them all
